packages/theme-chalk/src/message-box.scss

```diff
@@ -189,7 +189,7 @@
     }
 
     @include e((btns, content)) {
-      text-align: center;
+      justify-content: center;
```

Comment:
I think the content should also be centered.

Reply:
> I think the content should also be centered.

Yes, you are right. I have revised the code again

![image](https://user-images.githubusercontent.com/29992752/156172632-ef688b11-a616-4c37-964d-5f9cc1914a7e.png)